app/server/lib/FlexServer.ts

```diff
@@ -1948,6 +1933,14 @@ export class FlexServer implements GristServer {
     }));
   }
 
+  public addConfigEndpoints() {
```

Comment:
What is the reason for the Config Api ? Can't we just inline all the things here? It looks a little bit artificial.

Reply:
I already find this module to be very large and sprawling. It's 2.5k lines long. I wanted a little separation. I also liked using the static variable in a class for namespacing.

Reply:
Ok, but can you do this refactor later? For example, all things used by the admin panel are already grouped in `/api/admin` path, there are already some other endpoints created. So it makes more sense to move them all together.

Sorry for the problems that caused you to use `static` fields. I did a little digging, and found `devServerMain.ts`, when you read it, around line 131, we are starting a separate home server (a copy). Probably this is causing the two `API` problems.

This also mean, that the logic for restarting, needs to be moved out of the `FlexServer` instance, and somehow added at the entry point of the application (the script that is eventually run by docker), and somehow injected into FlexServer.

Other idea that I have, is to move this logic (that is built around this static variable), to the wrapper you created previously around the server. We can send and receive messages between them, so maybe it can hold the static state. Wdyt?

Reply:
I've removed the static variable, but I still want to avoid adding too many lines to the `FlexServer` module.

The restart now happens by having a child Node.js process signal to its `supervisor` Node.js that it wants to be restarted. At the moment in the default Docker image, each `supervisor` process only spawns one child, so I believe this should work out.